dir.props

```diff
@@ -2,6 +2,12 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Condition="Exists('..\dir.props')" Project="..\dir.props" />
 
+  <!-- We shipped assembly file version 4.6.x up until the end of rc3.  Version assembly as 
+        4.6.x to ensure compatability in Visual Studio for an in-place update. -->
```

Comment:
This comment implies the change is required for libraries that are in-place updated for VS (or Desktop) scenarios. CoreCLR does not have any such scenario - is the change still required @weshaggard @ericstj ?